cmd/dmsg-discovery/internal/store/redis.go

```diff
@@ -70,6 +70,12 @@ func (r *redisStore) SetEntry(ctx context.Context, entry *disc.Entry, timeout ti
 			return disc.ErrUnexpected
 		}
 	}
+	if entry.Client != nil {
+		err = r.client.SAdd("clients", entry.Static.Hex()).Err()
+		if err != nil {
+			return disc.ErrUnexpected
```

Comment:
As `disc.ErrUnexpected` doesn't contain a returned error, it wouldn't be seen anywhere in logs. Could you please add an error log in case of an error?